charts/backstage/values.yaml

```diff
@@ -152,6 +188,8 @@ upstream:
             name: dynamic-plugins-npmrc
             readOnly: true
             subPath: .npmrc
+          - mountPath: /opt/app-root/src/.npm/_cacache
+            name: npmcacache
```

Comment:
This addition provides a needed ephemeral writeable volume since the root filesystem has been updated to be readonly